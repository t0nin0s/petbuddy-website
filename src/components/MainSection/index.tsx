import React from 'react';
import styled from 'styled-components'
import Button from '../Button'
import { mediaQueries } from '../../mediaqueries';
const Logo = require('../../images/logo.jpg')

const Container = styled.div`
display: flex;
flex-direction: column;
padding:14rem 2rem 0 2rem;
background-blend-mode: darken;
background: linear-gradient(90deg,rgba(120,128,145, 0.34) 40%,#EBEBEB 0%)!important;
letter-spacing: 2px;
line-height: 1.8em;
  ${mediaQueries("medium")`
  display: flex;
  flex-direction: row;
  justify-content: space-around;
  `}
`
const LogoImage = styled.img`
align-self: center;
box-shadow: 2px 5px 10px #666;
margin-bottom: 3rem;
width: 70%;
  ${mediaQueries("tablet")`
    width: 60%;
  `}
  ${mediaQueries("medium")`
    width: 40%;
    margin-left: 2%;
    margin-top: -6%;
  `}
  ${mediaQueries("laptop")`
    margin-top: -8%;
  `}
  ${mediaQueries("xl")`
    width: 30%;
    margin-left: 10%;
  `}
`
const Content = styled.div`
${mediaQueries("tablet")`
    margin: 0 6rem;
  `}
${mediaQueries("laptop")`
    width: 40%;
  `}
`
const Header = styled.h3`
color: #d7a4ac!important;
`
const SubHeader = styled.h1`
`
const Services = styled.p`
color: #a28bbb;
font - weight: 900;
`
const Paragraph = styled.p`
color: #666;
`
const ButtonDiv = styled.div`
padding: 3rem 0 6rem 0;
font-size:18px;
`

export default (props: any) => {
  const { motto, title, services, blurb } = props;
  return (
    <Container >
      <LogoImage src={Logo} alt="logo" />
      <Content>
        <Header>{motto} </Header>
        <SubHeader>{title}</SubHeader>
        <Services>{services}</Services>
        <Paragraph >{blurb}<br /></Paragraph>
        <ButtonDiv>
          <Button>Contact Us</Button>
        </ButtonDiv>
      </Content>
    </Container>
  );
}

