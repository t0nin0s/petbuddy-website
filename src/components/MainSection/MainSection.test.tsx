import React from 'react'
import MainSection from './index'
import { render } from '@testing-library/react'

describe('MainSection component', () => {
  it('should render', () => {
    const wrapper = render(<MainSection />)
    expect(wrapper).toMatchSnapshot();
  })
  it('should output some text ', () => {
    const wrapper = 'WHERE YOUR PET BECOMES OUR BUDDY';
    expect(wrapper).toEqual('WHERE YOUR PET BECOMES OUR BUDDY')
  })
})