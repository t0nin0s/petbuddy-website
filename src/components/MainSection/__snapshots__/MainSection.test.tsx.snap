// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainSection component should render 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-gsTCUz DRchx"
      >
        <img
          alt="logo"
          class="sc-dlfnbm bfosiP"
          src="logo.jpg"
        />
        <div
          class="sc-hKgILt kVxmdc"
        >
          <h3
            class="sc-eCssSg cEeQki"
          >
             
          </h3>
          <h1
            class="sc-jSgupP eplsmV"
          />
          <p
            class="sc-gKsewC kHFPCQ"
          />
          <p
            class="sc-iBPRYJ bnFpGD"
          >
            <br />
          </p>
          <div
            class="sc-fubCfw byZGmj"
          >
            <button
              class="MuiButtonBase-root MuiButton-root MuiButton-contained sc-bdfBwQ jdtooK"
              tabindex="0"
              type="button"
            >
              <span
                class="MuiButton-label"
              >
                Contact Us
              </span>
              <span
                class="MuiTouchRipple-root"
              />
            </button>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-gsTCUz DRchx"
    >
      <img
        alt="logo"
        class="sc-dlfnbm bfosiP"
        src="logo.jpg"
      />
      <div
        class="sc-hKgILt kVxmdc"
      >
        <h3
          class="sc-eCssSg cEeQki"
        >
           
        </h3>
        <h1
          class="sc-jSgupP eplsmV"
        />
        <p
          class="sc-gKsewC kHFPCQ"
        />
        <p
          class="sc-iBPRYJ bnFpGD"
        >
          <br />
        </p>
        <div
          class="sc-fubCfw byZGmj"
        >
          <button
            class="MuiButtonBase-root MuiButton-root MuiButton-contained sc-bdfBwQ jdtooK"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-label"
            >
              Contact Us
            </span>
            <span
              class="MuiTouchRipple-root"
            />
          </button>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
